src/state/apis/zapper/zapperApi.ts

```diff
@@ -388,12 +388,27 @@ export const zapper = createApi({
               const appAccountOpportunities = appAccountBalance.products
                 .flatMap(({ assets }) => assets)
                 .map<ReadOnlyOpportunityType | undefined>(asset => {
-                  const stakedAmountCryptoBaseUnit =
-                    //  Liquidity Pool staking
-                    asset.tokens.find(token => token.metaType === 'supplied')?.balanceRaw ??
-                    // Single-sided staking
-                    asset.balanceRaw ??
-                    '0'
+                  const stakedAmountCryptoBaseUnitAccessor = (() => {
```

Comment:
kind of a strange var name imo. how about just `stakingAsset` or something like that? seems like there is a bit of overlap/mix with staking vs lp terminology which makes things a bit confusing

Reply:
Agree the naming is weird, though that was precisely to make https://github.com/shapeshift/web/pull/4679#discussion_r1223209493 less confusing - `stakingAsset` vs. `topLevelAsset` would be confusing, this makes it clear that this is to be used as a base unit staked amount accessor